pyproject.toml

```diff
@@ -22,7 +22,8 @@ redis = { version = "^4.2.0rc1", optional = true }
 aiomcache = { version = "*", optional = true }
 pendulum = "*"
 aiobotocore = { version = "^1.4.1", optional = true }
-typing-extensions = { version = ">=4.1.0", markers = "python_version < \"3.10\"" }
+types-aiobotocore = { extras = ["dynamodb"], version = "^2.5.0.post2", optional = true }
```

Comment:
Should the package `types-aiobotocore` for type hinting be placed under `[tool.poetry.group.dev.dependencies]`?